os/kernel/log_dump/log_dump.c

```diff
@@ -47,11 +51,14 @@
 #endif
 
 #define LOG_DUMP_OK			 0
+#define LOG_DUMP_OK_FSZ			 1
 #define LOG_DUMP_MEM_FAIL		-1
 #define LOG_DUMP_HEAP_FAIL		-2
 #define LOG_DUMP_SIZE_FAIL		-3
+#define LOG_DUMP_STR_FREEMEM		15
 
 #define LOG_CHUNK_SIZE			sizeof(struct log_dump_chunk_s)
+#define MINIZ_COMBOUND_SIZE		(CONFIG_LOG_DUMP_CHUNK_SIZE + 1024)
```

Comment:
Is `MINIZ` compression algorithm used for log_dump?
If not, is this needed?

Reply:
removed